meta.go

```diff
@@ -438,7 +442,57 @@ func main() {
 			},
 			Flags: []cli.Flag{jsonValueFlag},
 		},
+		{
+			Name:  "lastSuccessfulMeta",
```

Comment:
I would prefer if this logic can be combined to `--external` flag. external flag in `get` command, today just changes the `metaFile` and expects [launcher](https://github.com/screwdriver-cd/launcher/) to already populate the file. Logic of since command should go to handling behavior when the file doesn't exist. https://github.com/screwdriver-cd/meta-cli/blob/master/meta.go#L45-L52

Reply:
I think we shouldn't combine.
Since `--external` is more like `--external-trigger`, only works if the the external job is the triggering job.
Last successful meta is for any job.

Reply:
Yeah, that makes sense - please take a look at the latest - instead of alias for same flag, used cooperative flag to influence behavior:

```   --fetch-nonexistent-external, -E  When set, if metadata provided by --external flag is nonexistent, fetch lastSuccessfulMeta from the external job
```